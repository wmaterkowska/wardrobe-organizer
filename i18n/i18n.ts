import i18n from 'i18next';
import { initReactI18next } from 'react-i18next';
//import LanguageDetector from 'i18next-react-native-language-detector';

import en_common from './locales/en/common.json';
import en_about_app from './locales/en/AboutApp.json';
import en_navigation from './locales/en/navigation.json';
import en_insights from './locales/en/insights.json'

import pl_common from './locales/pl/common.json';
import pl_about_app from './locales/pl/AboutApp.json';
import pl_navigation from './locales/pl/navigation.json';
import pl_insights from './locales/pl/insights.json'

i18n
//  .use(LanguageDetector)
  .use(initReactI18next)
  .init({
    fallbackLng: 'en',
    compatibilityJSON: 'v3',
    ns: ['common', 'about_app', 'navigation', 'insights'],
    defaultNS: 'common',
    resources: {
      en: {
        common: en_common,
        about_app: en_about_app,
        navigation: en_navigation,
        insights: en_insights,
      },
      pl: {
        common: pl_common,
        about_app: pl_about_app,
        navigation: pl_navigation,
        insights: pl_insights,
      },
    },
    interpolation: {
      escapeValue: false,
    },
    pluralSeparator: '_'
  });

export default i18n;
