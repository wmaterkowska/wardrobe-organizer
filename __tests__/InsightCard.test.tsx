import React from 'react';
import { render } from '@testing-library/react-native';

import '../i18n/i18n';
import i18n from '../i18n/i18n';

import { Text } from 'react-native-paper';
import InsightCard from '../components/InsightCard';


describe('itemsInWardrobe Insight Card', () => {
  it('renders correctly english with count = 1', () => {
    i18n.changeLanguage('en');
    const { getByText } = render(
      <InsightCard type={'itemsInWardrobe'} data={1}/>
    );
    expect(getByText('You have 1 piece in your wardrobe.')).toBeTruthy();
  });

  it('renders correctly english with count = 5', () => {
    i18n.changeLanguage('en');
    const { getByText } = render(
      <InsightCard type={'itemsInWardrobe'} data={5} />
    );
    expect(getByText('You have 5 pieces in your wardrobe.')).toBeTruthy();
  });

  it('renders correctly polish with count = 1', () => {
    i18n.changeLanguage('pl');
    const { getByText } = render(
      <InsightCard type={'itemsInWardrobe'} data={1}/>
    );
    expect(getByText('Masz 1 przedmiot w swojej szafie.')).toBeTruthy();
  });

  it('renders correctly polish with count = 2', () => {
    i18n.changeLanguage('pl');
    const { getByText } = render(
      <InsightCard type={'itemsInWardrobe'} data={2} />
    );
    expect(getByText('Masz 2 przedmioty w swojej szafie.')).toBeTruthy();
  });

  it('renders correctly polish with count = 5', () => {
    i18n.changeLanguage('pl');
    const { getByText } = render(
      <InsightCard type={'itemsInWardrobe'} data={5}/>
    );
    expect(getByText('Masz 5 przedmiotów w swojej szafie.')).toBeTruthy();
  });

  it('renders correctly polish with count = 22', () => {
    i18n.changeLanguage('pl');
    const { getByText } = render(
      <InsightCard type={'itemsInWardrobe'} data={22} />
    );
    expect(getByText('Masz 22 przedmioty w swojej szafie.')).toBeTruthy();
  });

  it('renders correctly polish with count = 25', () => {
    i18n.changeLanguage('pl');
    const { getByText } = render(
      <InsightCard type={'itemsInWardrobe'} data={25}/>
    );
    expect(getByText('Masz 25 przedmiotów w swojej szafie.')).toBeTruthy();
  });

  it('renders correctly polish with count = 0', () => {
    i18n.changeLanguage('pl');
    const { getByText } = render(
      <InsightCard type={'itemsInWardrobe'} data={0} />
    );
    expect(getByText('Masz 0 przedmiotów w swojej szafie.')).toBeTruthy();
  });

});
