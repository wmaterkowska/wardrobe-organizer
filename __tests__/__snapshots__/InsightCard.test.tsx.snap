// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`itemsInWardrobe Insight Card renders correctly english with count = 1 1`] = `
<View
  collapsable={false}
  style={
    {
      "backgroundColor": "transparent",
      "borderRadius": 0,
      "borderTopLeftRadius": 20,
      "margin": 0,
      "shadowColor": "#000",
      "shadowOffset": {
        "height": 0,
        "width": 0,
      },
      "shadowOpacity": 0,
      "shadowRadius": 0,
      "width": "100%",
    }
  }
  testID="card-container-outer-layer"
>
  <View
    collapsable={false}
    style={
      {
        "backgroundColor": "transparent",
        "borderColor": "transparent",
        "borderRadius": 0,
        "borderTopLeftRadius": 20,
        "borderWidth": 0,
        "flex": undefined,
        "padding": 0,
        "shadowColor": "#000",
        "shadowOffset": {
          "height": 0,
          "width": 0,
        },
        "shadowOpacity": 0,
        "shadowRadius": 0,
      }
    }
    testID="card-container"
  >
    <View
      pointerEvents="none"
      style={
        [
          {
            "borderColor": "transparent",
          },
          {
            "borderWidth": 1,
            "height": "100%",
            "position": "absolute",
            "width": "100%",
            "zIndex": 2,
          },
          {
            "borderRadius": 0,
            "borderTopLeftRadius": 20,
          },
        ]
      }
      testID="card-outline"
    />
    <View
      style={
        [
          {
            "flexShrink": 1,
          },
          undefined,
        ]
      }
      testID="card"
    >
      <View
        borderRadiusStyles={
          {
            "borderRadius": 0,
            "borderTopLeftRadius": 20,
          }
        }
        style={
          [
            {
              "paddingHorizontal": 16,
            },
            {
              "paddingVertical": 16,
            },
            {
              "marginBottom": 20,
              "marginLeft": 16,
              "marginTop": 12,
            },
          ]
        }
      >
        <Text
          style={
            [
              {
                "textAlign": "left",
              },
              {
                "color": "rgba(28, 27, 31, 1)",
                "writingDirection": "ltr",
              },
              [
                {
                  "fontFamily": "System",
                  "fontSize": 16,
                  "fontWeight": "400",
                  "letterSpacing": 0.15,
                  "lineHeight": 24,
                },
                {
                  "color": "rgba(255, 255, 255, 1)",
                  "fontWeight": "600",
                  "marginBottom": 6,
                },
              ],
            ]
          }
        >
          You have 
          <Text
            style={
              [
                {
                  "textAlign": "left",
                },
                {
                  "color": "rgba(28, 27, 31, 1)",
                  "writingDirection": "ltr",
                },
                [
                  {
                    "fontFamily": "System",
                    "fontSize": 12,
                    "fontWeight": "400",
                    "letterSpacing": 0.4,
                    "lineHeight": 16,
                  },
                  {
                    "color": "rgba(255, 255, 255, 1)",
                    "fontWeight": "bold",
                    "marginBottom": 12,
                  },
                ],
              ]
            }
          >
            1
          </Text>
           piece in your wardrobe.
        </Text>
      </View>
    </View>
  </View>
</View>
`;

exports[`itemsInWardrobe Insight Card renders correctly english with count = 5 1`] = `
<View
  collapsable={false}
  style={
    {
      "backgroundColor": "transparent",
      "borderRadius": 0,
      "borderTopLeftRadius": 20,
      "margin": 0,
      "shadowColor": "#000",
      "shadowOffset": {
        "height": 0,
        "width": 0,
      },
      "shadowOpacity": 0,
      "shadowRadius": 0,
      "width": "100%",
    }
  }
  testID="card-container-outer-layer"
>
  <View
    collapsable={false}
    style={
      {
        "backgroundColor": "transparent",
        "borderColor": "transparent",
        "borderRadius": 0,
        "borderTopLeftRadius": 20,
        "borderWidth": 0,
        "flex": undefined,
        "padding": 0,
        "shadowColor": "#000",
        "shadowOffset": {
          "height": 0,
          "width": 0,
        },
        "shadowOpacity": 0,
        "shadowRadius": 0,
      }
    }
    testID="card-container"
  >
    <View
      pointerEvents="none"
      style={
        [
          {
            "borderColor": "transparent",
          },
          {
            "borderWidth": 1,
            "height": "100%",
            "position": "absolute",
            "width": "100%",
            "zIndex": 2,
          },
          {
            "borderRadius": 0,
            "borderTopLeftRadius": 20,
          },
        ]
      }
      testID="card-outline"
    />
    <View
      style={
        [
          {
            "flexShrink": 1,
          },
          undefined,
        ]
      }
      testID="card"
    >
      <View
        borderRadiusStyles={
          {
            "borderRadius": 0,
            "borderTopLeftRadius": 20,
          }
        }
        style={
          [
            {
              "paddingHorizontal": 16,
            },
            {
              "paddingVertical": 16,
            },
            {
              "marginBottom": 20,
              "marginLeft": 16,
              "marginTop": 12,
            },
          ]
        }
      >
        <Text
          style={
            [
              {
                "textAlign": "left",
              },
              {
                "color": "rgba(28, 27, 31, 1)",
                "writingDirection": "ltr",
              },
              [
                {
                  "fontFamily": "System",
                  "fontSize": 16,
                  "fontWeight": "400",
                  "letterSpacing": 0.15,
                  "lineHeight": 24,
                },
                {
                  "color": "rgba(255, 255, 255, 1)",
                  "fontWeight": "600",
                  "marginBottom": 6,
                },
              ],
            ]
          }
        >
          You have 
          <Text
            style={
              [
                {
                  "textAlign": "left",
                },
                {
                  "color": "rgba(28, 27, 31, 1)",
                  "writingDirection": "ltr",
                },
                [
                  {
                    "fontFamily": "System",
                    "fontSize": 12,
                    "fontWeight": "400",
                    "letterSpacing": 0.4,
                    "lineHeight": 16,
                  },
                  {
                    "color": "rgba(255, 255, 255, 1)",
                    "fontWeight": "bold",
                    "marginBottom": 12,
                  },
                ],
              ]
            }
          >
            5
          </Text>
           pieces in your wardrobe.
        </Text>
      </View>
    </View>
  </View>
</View>
`;

exports[`itemsInWardrobe Insight Card renders correctly polish with count = 0 1`] = `
<View
  collapsable={false}
  style={
    {
      "backgroundColor": "transparent",
      "borderRadius": 0,
      "borderTopLeftRadius": 20,
      "margin": 0,
      "shadowColor": "#000",
      "shadowOffset": {
        "height": 0,
        "width": 0,
      },
      "shadowOpacity": 0,
      "shadowRadius": 0,
      "width": "100%",
    }
  }
  testID="card-container-outer-layer"
>
  <View
    collapsable={false}
    style={
      {
        "backgroundColor": "transparent",
        "borderColor": "transparent",
        "borderRadius": 0,
        "borderTopLeftRadius": 20,
        "borderWidth": 0,
        "flex": undefined,
        "padding": 0,
        "shadowColor": "#000",
        "shadowOffset": {
          "height": 0,
          "width": 0,
        },
        "shadowOpacity": 0,
        "shadowRadius": 0,
      }
    }
    testID="card-container"
  >
    <View
      pointerEvents="none"
      style={
        [
          {
            "borderColor": "transparent",
          },
          {
            "borderWidth": 1,
            "height": "100%",
            "position": "absolute",
            "width": "100%",
            "zIndex": 2,
          },
          {
            "borderRadius": 0,
            "borderTopLeftRadius": 20,
          },
        ]
      }
      testID="card-outline"
    />
    <View
      style={
        [
          {
            "flexShrink": 1,
          },
          undefined,
        ]
      }
      testID="card"
    >
      <View
        borderRadiusStyles={
          {
            "borderRadius": 0,
            "borderTopLeftRadius": 20,
          }
        }
        style={
          [
            {
              "paddingHorizontal": 16,
            },
            {
              "paddingVertical": 16,
            },
            {
              "marginBottom": 20,
              "marginLeft": 16,
              "marginTop": 12,
            },
          ]
        }
      >
        <Text
          style={
            [
              {
                "textAlign": "left",
              },
              {
                "color": "rgba(28, 27, 31, 1)",
                "writingDirection": "ltr",
              },
              [
                {
                  "fontFamily": "System",
                  "fontSize": 16,
                  "fontWeight": "400",
                  "letterSpacing": 0.15,
                  "lineHeight": 24,
                },
                {
                  "color": "rgba(255, 255, 255, 1)",
                  "fontWeight": "600",
                  "marginBottom": 6,
                },
              ],
            ]
          }
        >
          You have 
          <Text
            style={
              [
                {
                  "textAlign": "left",
                },
                {
                  "color": "rgba(28, 27, 31, 1)",
                  "writingDirection": "ltr",
                },
                [
                  {
                    "fontFamily": "System",
                    "fontSize": 12,
                    "fontWeight": "400",
                    "letterSpacing": 0.4,
                    "lineHeight": 16,
                  },
                  {
                    "color": "rgba(255, 255, 255, 1)",
                    "fontWeight": "bold",
                    "marginBottom": 12,
                  },
                ],
              ]
            }
          >
            5
          </Text>
           pieces in your wardrobe.
        </Text>
      </View>
    </View>
  </View>
</View>
`;

exports[`itemsInWardrobe Insight Card renders correctly polish with count = 1 1`] = `
<View
  collapsable={false}
  style={
    {
      "backgroundColor": "transparent",
      "borderRadius": 0,
      "borderTopLeftRadius": 20,
      "margin": 0,
      "shadowColor": "#000",
      "shadowOffset": {
        "height": 0,
        "width": 0,
      },
      "shadowOpacity": 0,
      "shadowRadius": 0,
      "width": "100%",
    }
  }
  testID="card-container-outer-layer"
>
  <View
    collapsable={false}
    style={
      {
        "backgroundColor": "transparent",
        "borderColor": "transparent",
        "borderRadius": 0,
        "borderTopLeftRadius": 20,
        "borderWidth": 0,
        "flex": undefined,
        "padding": 0,
        "shadowColor": "#000",
        "shadowOffset": {
          "height": 0,
          "width": 0,
        },
        "shadowOpacity": 0,
        "shadowRadius": 0,
      }
    }
    testID="card-container"
  >
    <View
      pointerEvents="none"
      style={
        [
          {
            "borderColor": "transparent",
          },
          {
            "borderWidth": 1,
            "height": "100%",
            "position": "absolute",
            "width": "100%",
            "zIndex": 2,
          },
          {
            "borderRadius": 0,
            "borderTopLeftRadius": 20,
          },
        ]
      }
      testID="card-outline"
    />
    <View
      style={
        [
          {
            "flexShrink": 1,
          },
          undefined,
        ]
      }
      testID="card"
    >
      <View
        borderRadiusStyles={
          {
            "borderRadius": 0,
            "borderTopLeftRadius": 20,
          }
        }
        style={
          [
            {
              "paddingHorizontal": 16,
            },
            {
              "paddingVertical": 16,
            },
            {
              "marginBottom": 20,
              "marginLeft": 16,
              "marginTop": 12,
            },
          ]
        }
      >
        <Text
          style={
            [
              {
                "textAlign": "left",
              },
              {
                "color": "rgba(28, 27, 31, 1)",
                "writingDirection": "ltr",
              },
              [
                {
                  "fontFamily": "System",
                  "fontSize": 16,
                  "fontWeight": "400",
                  "letterSpacing": 0.15,
                  "lineHeight": 24,
                },
                {
                  "color": "rgba(255, 255, 255, 1)",
                  "fontWeight": "600",
                  "marginBottom": 6,
                },
              ],
            ]
          }
        >
          You have 
          <Text
            style={
              [
                {
                  "textAlign": "left",
                },
                {
                  "color": "rgba(28, 27, 31, 1)",
                  "writingDirection": "ltr",
                },
                [
                  {
                    "fontFamily": "System",
                    "fontSize": 12,
                    "fontWeight": "400",
                    "letterSpacing": 0.4,
                    "lineHeight": 16,
                  },
                  {
                    "color": "rgba(255, 255, 255, 1)",
                    "fontWeight": "bold",
                    "marginBottom": 12,
                  },
                ],
              ]
            }
          >
            1
          </Text>
           piece in your wardrobe.
        </Text>
      </View>
    </View>
  </View>
</View>
`;

exports[`itemsInWardrobe Insight Card renders correctly polish with count = 2 1`] = `
<View
  collapsable={false}
  style={
    {
      "backgroundColor": "transparent",
      "borderRadius": 0,
      "borderTopLeftRadius": 20,
      "margin": 0,
      "shadowColor": "#000",
      "shadowOffset": {
        "height": 0,
        "width": 0,
      },
      "shadowOpacity": 0,
      "shadowRadius": 0,
      "width": "100%",
    }
  }
  testID="card-container-outer-layer"
>
  <View
    collapsable={false}
    style={
      {
        "backgroundColor": "transparent",
        "borderColor": "transparent",
        "borderRadius": 0,
        "borderTopLeftRadius": 20,
        "borderWidth": 0,
        "flex": undefined,
        "padding": 0,
        "shadowColor": "#000",
        "shadowOffset": {
          "height": 0,
          "width": 0,
        },
        "shadowOpacity": 0,
        "shadowRadius": 0,
      }
    }
    testID="card-container"
  >
    <View
      pointerEvents="none"
      style={
        [
          {
            "borderColor": "transparent",
          },
          {
            "borderWidth": 1,
            "height": "100%",
            "position": "absolute",
            "width": "100%",
            "zIndex": 2,
          },
          {
            "borderRadius": 0,
            "borderTopLeftRadius": 20,
          },
        ]
      }
      testID="card-outline"
    />
    <View
      style={
        [
          {
            "flexShrink": 1,
          },
          undefined,
        ]
      }
      testID="card"
    >
      <View
        borderRadiusStyles={
          {
            "borderRadius": 0,
            "borderTopLeftRadius": 20,
          }
        }
        style={
          [
            {
              "paddingHorizontal": 16,
            },
            {
              "paddingVertical": 16,
            },
            {
              "marginBottom": 20,
              "marginLeft": 16,
              "marginTop": 12,
            },
          ]
        }
      >
        <Text
          style={
            [
              {
                "textAlign": "left",
              },
              {
                "color": "rgba(28, 27, 31, 1)",
                "writingDirection": "ltr",
              },
              [
                {
                  "fontFamily": "System",
                  "fontSize": 16,
                  "fontWeight": "400",
                  "letterSpacing": 0.15,
                  "lineHeight": 24,
                },
                {
                  "color": "rgba(255, 255, 255, 1)",
                  "fontWeight": "600",
                  "marginBottom": 6,
                },
              ],
            ]
          }
        >
          You have 
          <Text
            style={
              [
                {
                  "textAlign": "left",
                },
                {
                  "color": "rgba(28, 27, 31, 1)",
                  "writingDirection": "ltr",
                },
                [
                  {
                    "fontFamily": "System",
                    "fontSize": 12,
                    "fontWeight": "400",
                    "letterSpacing": 0.4,
                    "lineHeight": 16,
                  },
                  {
                    "color": "rgba(255, 255, 255, 1)",
                    "fontWeight": "bold",
                    "marginBottom": 12,
                  },
                ],
              ]
            }
          >
            5
          </Text>
           pieces in your wardrobe.
        </Text>
      </View>
    </View>
  </View>
</View>
`;

exports[`itemsInWardrobe Insight Card renders correctly polish with count = 5 1`] = `
<View
  collapsable={false}
  style={
    {
      "backgroundColor": "transparent",
      "borderRadius": 0,
      "borderTopLeftRadius": 20,
      "margin": 0,
      "shadowColor": "#000",
      "shadowOffset": {
        "height": 0,
        "width": 0,
      },
      "shadowOpacity": 0,
      "shadowRadius": 0,
      "width": "100%",
    }
  }
  testID="card-container-outer-layer"
>
  <View
    collapsable={false}
    style={
      {
        "backgroundColor": "transparent",
        "borderColor": "transparent",
        "borderRadius": 0,
        "borderTopLeftRadius": 20,
        "borderWidth": 0,
        "flex": undefined,
        "padding": 0,
        "shadowColor": "#000",
        "shadowOffset": {
          "height": 0,
          "width": 0,
        },
        "shadowOpacity": 0,
        "shadowRadius": 0,
      }
    }
    testID="card-container"
  >
    <View
      pointerEvents="none"
      style={
        [
          {
            "borderColor": "transparent",
          },
          {
            "borderWidth": 1,
            "height": "100%",
            "position": "absolute",
            "width": "100%",
            "zIndex": 2,
          },
          {
            "borderRadius": 0,
            "borderTopLeftRadius": 20,
          },
        ]
      }
      testID="card-outline"
    />
    <View
      style={
        [
          {
            "flexShrink": 1,
          },
          undefined,
        ]
      }
      testID="card"
    >
      <View
        borderRadiusStyles={
          {
            "borderRadius": 0,
            "borderTopLeftRadius": 20,
          }
        }
        style={
          [
            {
              "paddingHorizontal": 16,
            },
            {
              "paddingVertical": 16,
            },
            {
              "marginBottom": 20,
              "marginLeft": 16,
              "marginTop": 12,
            },
          ]
        }
      >
        <Text
          style={
            [
              {
                "textAlign": "left",
              },
              {
                "color": "rgba(28, 27, 31, 1)",
                "writingDirection": "ltr",
              },
              [
                {
                  "fontFamily": "System",
                  "fontSize": 16,
                  "fontWeight": "400",
                  "letterSpacing": 0.15,
                  "lineHeight": 24,
                },
                {
                  "color": "rgba(255, 255, 255, 1)",
                  "fontWeight": "600",
                  "marginBottom": 6,
                },
              ],
            ]
          }
        >
          You have 
          <Text
            style={
              [
                {
                  "textAlign": "left",
                },
                {
                  "color": "rgba(28, 27, 31, 1)",
                  "writingDirection": "ltr",
                },
                [
                  {
                    "fontFamily": "System",
                    "fontSize": 12,
                    "fontWeight": "400",
                    "letterSpacing": 0.4,
                    "lineHeight": 16,
                  },
                  {
                    "color": "rgba(255, 255, 255, 1)",
                    "fontWeight": "bold",
                    "marginBottom": 12,
                  },
                ],
              ]
            }
          >
            1
          </Text>
           piece in your wardrobe.
        </Text>
      </View>
    </View>
  </View>
</View>
`;

exports[`itemsInWardrobe Insight Card renders correctly polish with count = 22 1`] = `
<View
  collapsable={false}
  style={
    {
      "backgroundColor": "transparent",
      "borderRadius": 0,
      "borderTopLeftRadius": 20,
      "margin": 0,
      "shadowColor": "#000",
      "shadowOffset": {
        "height": 0,
        "width": 0,
      },
      "shadowOpacity": 0,
      "shadowRadius": 0,
      "width": "100%",
    }
  }
  testID="card-container-outer-layer"
>
  <View
    collapsable={false}
    style={
      {
        "backgroundColor": "transparent",
        "borderColor": "transparent",
        "borderRadius": 0,
        "borderTopLeftRadius": 20,
        "borderWidth": 0,
        "flex": undefined,
        "padding": 0,
        "shadowColor": "#000",
        "shadowOffset": {
          "height": 0,
          "width": 0,
        },
        "shadowOpacity": 0,
        "shadowRadius": 0,
      }
    }
    testID="card-container"
  >
    <View
      pointerEvents="none"
      style={
        [
          {
            "borderColor": "transparent",
          },
          {
            "borderWidth": 1,
            "height": "100%",
            "position": "absolute",
            "width": "100%",
            "zIndex": 2,
          },
          {
            "borderRadius": 0,
            "borderTopLeftRadius": 20,
          },
        ]
      }
      testID="card-outline"
    />
    <View
      style={
        [
          {
            "flexShrink": 1,
          },
          undefined,
        ]
      }
      testID="card"
    >
      <View
        borderRadiusStyles={
          {
            "borderRadius": 0,
            "borderTopLeftRadius": 20,
          }
        }
        style={
          [
            {
              "paddingHorizontal": 16,
            },
            {
              "paddingVertical": 16,
            },
            {
              "marginBottom": 20,
              "marginLeft": 16,
              "marginTop": 12,
            },
          ]
        }
      >
        <Text
          style={
            [
              {
                "textAlign": "left",
              },
              {
                "color": "rgba(28, 27, 31, 1)",
                "writingDirection": "ltr",
              },
              [
                {
                  "fontFamily": "System",
                  "fontSize": 16,
                  "fontWeight": "400",
                  "letterSpacing": 0.15,
                  "lineHeight": 24,
                },
                {
                  "color": "rgba(255, 255, 255, 1)",
                  "fontWeight": "600",
                  "marginBottom": 6,
                },
              ],
            ]
          }
        >
          You have 
          <Text
            style={
              [
                {
                  "textAlign": "left",
                },
                {
                  "color": "rgba(28, 27, 31, 1)",
                  "writingDirection": "ltr",
                },
                [
                  {
                    "fontFamily": "System",
                    "fontSize": 12,
                    "fontWeight": "400",
                    "letterSpacing": 0.4,
                    "lineHeight": 16,
                  },
                  {
                    "color": "rgba(255, 255, 255, 1)",
                    "fontWeight": "bold",
                    "marginBottom": 12,
                  },
                ],
              ]
            }
          >
            5
          </Text>
           pieces in your wardrobe.
        </Text>
      </View>
    </View>
  </View>
</View>
`;

exports[`itemsInWardrobe Insight Card renders correctly polish with count = 25 1`] = `
<View
  collapsable={false}
  style={
    {
      "backgroundColor": "transparent",
      "borderRadius": 0,
      "borderTopLeftRadius": 20,
      "margin": 0,
      "shadowColor": "#000",
      "shadowOffset": {
        "height": 0,
        "width": 0,
      },
      "shadowOpacity": 0,
      "shadowRadius": 0,
      "width": "100%",
    }
  }
  testID="card-container-outer-layer"
>
  <View
    collapsable={false}
    style={
      {
        "backgroundColor": "transparent",
        "borderColor": "transparent",
        "borderRadius": 0,
        "borderTopLeftRadius": 20,
        "borderWidth": 0,
        "flex": undefined,
        "padding": 0,
        "shadowColor": "#000",
        "shadowOffset": {
          "height": 0,
          "width": 0,
        },
        "shadowOpacity": 0,
        "shadowRadius": 0,
      }
    }
    testID="card-container"
  >
    <View
      pointerEvents="none"
      style={
        [
          {
            "borderColor": "transparent",
          },
          {
            "borderWidth": 1,
            "height": "100%",
            "position": "absolute",
            "width": "100%",
            "zIndex": 2,
          },
          {
            "borderRadius": 0,
            "borderTopLeftRadius": 20,
          },
        ]
      }
      testID="card-outline"
    />
    <View
      style={
        [
          {
            "flexShrink": 1,
          },
          undefined,
        ]
      }
      testID="card"
    >
      <View
        borderRadiusStyles={
          {
            "borderRadius": 0,
            "borderTopLeftRadius": 20,
          }
        }
        style={
          [
            {
              "paddingHorizontal": 16,
            },
            {
              "paddingVertical": 16,
            },
            {
              "marginBottom": 20,
              "marginLeft": 16,
              "marginTop": 12,
            },
          ]
        }
      >
        <Text
          style={
            [
              {
                "textAlign": "left",
              },
              {
                "color": "rgba(28, 27, 31, 1)",
                "writingDirection": "ltr",
              },
              [
                {
                  "fontFamily": "System",
                  "fontSize": 16,
                  "fontWeight": "400",
                  "letterSpacing": 0.15,
                  "lineHeight": 24,
                },
                {
                  "color": "rgba(255, 255, 255, 1)",
                  "fontWeight": "600",
                  "marginBottom": 6,
                },
              ],
            ]
          }
        >
          You have 
          <Text
            style={
              [
                {
                  "textAlign": "left",
                },
                {
                  "color": "rgba(28, 27, 31, 1)",
                  "writingDirection": "ltr",
                },
                [
                  {
                    "fontFamily": "System",
                    "fontSize": 12,
                    "fontWeight": "400",
                    "letterSpacing": 0.4,
                    "lineHeight": 16,
                  },
                  {
                    "color": "rgba(255, 255, 255, 1)",
                    "fontWeight": "bold",
                    "marginBottom": 12,
                  },
                ],
              ]
            }
          >
            1
          </Text>
           piece in your wardrobe.
        </Text>
      </View>
    </View>
  </View>
</View>
`;

exports[`itemsInWardrobe Insight Card renders correctly with count = 1 1`] = `
<View
  collapsable={false}
  style={
    {
      "backgroundColor": "transparent",
      "borderRadius": 0,
      "borderTopLeftRadius": 20,
      "margin": 0,
      "shadowColor": "#000",
      "shadowOffset": {
        "height": 0,
        "width": 0,
      },
      "shadowOpacity": 0,
      "shadowRadius": 0,
      "width": "100%",
    }
  }
  testID="card-container-outer-layer"
>
  <View
    collapsable={false}
    style={
      {
        "backgroundColor": "transparent",
        "borderColor": "transparent",
        "borderRadius": 0,
        "borderTopLeftRadius": 20,
        "borderWidth": 0,
        "flex": undefined,
        "padding": 0,
        "shadowColor": "#000",
        "shadowOffset": {
          "height": 0,
          "width": 0,
        },
        "shadowOpacity": 0,
        "shadowRadius": 0,
      }
    }
    testID="card-container"
  >
    <View
      pointerEvents="none"
      style={
        [
          {
            "borderColor": "transparent",
          },
          {
            "borderWidth": 1,
            "height": "100%",
            "position": "absolute",
            "width": "100%",
            "zIndex": 2,
          },
          {
            "borderRadius": 0,
            "borderTopLeftRadius": 20,
          },
        ]
      }
      testID="card-outline"
    />
    <View
      style={
        [
          {
            "flexShrink": 1,
          },
          undefined,
        ]
      }
      testID="card"
    >
      <View
        borderRadiusStyles={
          {
            "borderRadius": 0,
            "borderTopLeftRadius": 20,
          }
        }
        style={
          [
            {
              "paddingHorizontal": 16,
            },
            {
              "paddingVertical": 16,
            },
            {
              "marginBottom": 20,
              "marginLeft": 16,
              "marginTop": 12,
            },
          ]
        }
      >
        <Text
          style={
            [
              {
                "textAlign": "left",
              },
              {
                "color": "rgba(28, 27, 31, 1)",
                "writingDirection": "ltr",
              },
              [
                {
                  "fontFamily": "System",
                  "fontSize": 16,
                  "fontWeight": "400",
                  "letterSpacing": 0.15,
                  "lineHeight": 24,
                },
                {
                  "color": "rgba(255, 255, 255, 1)",
                  "fontWeight": "600",
                  "marginBottom": 6,
                },
              ],
            ]
          }
        >
          You have 
          <Text
            style={
              [
                {
                  "textAlign": "left",
                },
                {
                  "color": "rgba(28, 27, 31, 1)",
                  "writingDirection": "ltr",
                },
                [
                  {
                    "fontFamily": "System",
                    "fontSize": 12,
                    "fontWeight": "400",
                    "letterSpacing": 0.4,
                    "lineHeight": 16,
                  },
                  {
                    "color": "rgba(255, 255, 255, 1)",
                    "fontWeight": "bold",
                    "marginBottom": 12,
                  },
                ],
              ]
            }
          >
            1
          </Text>
           piece in your wardrobe.
        </Text>
      </View>
    </View>
  </View>
</View>
`;

exports[`itemsInWardrobe Insight Card renders correctly with count = 5 1`] = `
<View
  collapsable={false}
  style={
    {
      "backgroundColor": "transparent",
      "borderRadius": 0,
      "borderTopLeftRadius": 20,
      "margin": 0,
      "shadowColor": "#000",
      "shadowOffset": {
        "height": 0,
        "width": 0,
      },
      "shadowOpacity": 0,
      "shadowRadius": 0,
      "width": "100%",
    }
  }
  testID="card-container-outer-layer"
>
  <View
    collapsable={false}
    style={
      {
        "backgroundColor": "transparent",
        "borderColor": "transparent",
        "borderRadius": 0,
        "borderTopLeftRadius": 20,
        "borderWidth": 0,
        "flex": undefined,
        "padding": 0,
        "shadowColor": "#000",
        "shadowOffset": {
          "height": 0,
          "width": 0,
        },
        "shadowOpacity": 0,
        "shadowRadius": 0,
      }
    }
    testID="card-container"
  >
    <View
      pointerEvents="none"
      style={
        [
          {
            "borderColor": "transparent",
          },
          {
            "borderWidth": 1,
            "height": "100%",
            "position": "absolute",
            "width": "100%",
            "zIndex": 2,
          },
          {
            "borderRadius": 0,
            "borderTopLeftRadius": 20,
          },
        ]
      }
      testID="card-outline"
    />
    <View
      style={
        [
          {
            "flexShrink": 1,
          },
          undefined,
        ]
      }
      testID="card"
    >
      <View
        borderRadiusStyles={
          {
            "borderRadius": 0,
            "borderTopLeftRadius": 20,
          }
        }
        style={
          [
            {
              "paddingHorizontal": 16,
            },
            {
              "paddingVertical": 16,
            },
            {
              "marginBottom": 20,
              "marginLeft": 16,
              "marginTop": 12,
            },
          ]
        }
      >
        <Text
          style={
            [
              {
                "textAlign": "left",
              },
              {
                "color": "rgba(28, 27, 31, 1)",
                "writingDirection": "ltr",
              },
              [
                {
                  "fontFamily": "System",
                  "fontSize": 16,
                  "fontWeight": "400",
                  "letterSpacing": 0.15,
                  "lineHeight": 24,
                },
                {
                  "color": "rgba(255, 255, 255, 1)",
                  "fontWeight": "600",
                  "marginBottom": 6,
                },
              ],
            ]
          }
        >
          You have 
          <Text
            style={
              [
                {
                  "textAlign": "left",
                },
                {
                  "color": "rgba(28, 27, 31, 1)",
                  "writingDirection": "ltr",
                },
                [
                  {
                    "fontFamily": "System",
                    "fontSize": 12,
                    "fontWeight": "400",
                    "letterSpacing": 0.4,
                    "lineHeight": 16,
                  },
                  {
                    "color": "rgba(255, 255, 255, 1)",
                    "fontWeight": "bold",
                    "marginBottom": 12,
                  },
                ],
              ]
            }
          >
            5
          </Text>
           pieces in your wardrobe.
        </Text>
      </View>
    </View>
  </View>
</View>
`;
